specs/electra/fork.md

```diff
@@ -72,28 +72,7 @@ an irregular state change is made to upgrade to Electra.
 ```python
 def upgrade_to_electra(pre: deneb.BeaconState) -> BeaconState:
     epoch = deneb.get_current_epoch(pre)
-    latest_execution_payload_header = ExecutionPayloadHeader(
-        parent_hash=pre.latest_execution_payload_header.parent_hash,
-        fee_recipient=pre.latest_execution_payload_header.fee_recipient,
-        state_root=pre.latest_execution_payload_header.state_root,
-        receipts_root=pre.latest_execution_payload_header.receipts_root,
-        logs_bloom=pre.latest_execution_payload_header.logs_bloom,
-        prev_randao=pre.latest_execution_payload_header.prev_randao,
-        block_number=pre.latest_execution_payload_header.block_number,
-        gas_limit=pre.latest_execution_payload_header.gas_limit,
-        gas_used=pre.latest_execution_payload_header.gas_used,
-        timestamp=pre.latest_execution_payload_header.timestamp,
-        extra_data=pre.latest_execution_payload_header.extra_data,
-        base_fee_per_gas=pre.latest_execution_payload_header.base_fee_per_gas,
-        block_hash=pre.latest_execution_payload_header.block_hash,
-        transactions_root=pre.latest_execution_payload_header.transactions_root,
-        withdrawals_root=pre.latest_execution_payload_header.withdrawals_root,
-        blob_gas_used=pre.latest_execution_payload_header.blob_gas_used,
-        excess_blob_gas=pre.latest_execution_payload_header.excess_blob_gas,
-        deposit_requests_root=Root(),  # [New in Electra:EIP6110]
```

Comment:
what do we think about mirroring the `requests_root` here like in the EL block header?

Reply:
I personally see no use case for it on CL side. Those CL clients that utilize RLP implementations and run `block_hash` check during optimistic sync on their own will have to compute `requests_root` regardless on whether it is contained by the payload or not. Also we do not keep MPT root for e.g. transactions and withdrawals on CL side, there seems to be no reason to have it for the requests either.